### PR TITLE
Add example configuring alternate SPI ports.

### PR DESCRIPTION
**Pull request by nseidle**
Nothing fancy. Just another example to show how to configure additional ports.

### CODE REVIEW

File: libraries/SPI/examples/Example2_MoreSPIPorts/Example2_MoreSPIPorts.ino [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+#define SPI_SPEED 1000000
+#define SPI_ORDER MSBFIRST
+#define SPI_MODE SPI_MODE0
+
+SPISettings mySettings(SPI_SPEED, SPI_ORDER, SPI_MODE);
+
+const char *msg = "Hello world!";
+
+//SPIClass SPI(); //This is default and automatically defined on RedBoard/ATP/Nano. Uses pads 5/6/7 (SCK/MISO/MOSI).
```
Comment (oclyke) [inline libraries/SPI/examples/Example2_MoreSPIPorts/Example2_MoreSPIPorts.ino]: Just a nit: you are correctly using the pad vs pin notation but perhaps a user would not understand that pads 5/6/7 are actually (13, 12, 11) on the RedBoard (w/ Uno pin naming scheme). 

Should there be a note explaining this in this example?

**Comment by oclyke** [comment]
Oops accidentally chose 'request changes' when 'comment' would have been more appropriate.

**Comment by nseidle** [comment]
Agreed. I added a few comments. It's hard not to overwhelm the user with a breakdown of pads vs. pins which I don't think should be in the example. Let me know if you think it needs more.

**Comment by oclyke** [comment]
I think that does it nicely.

**Pull request merged**